docs/book/user-guide/best-practices/quick-wins.md

```diff
@@ -447,3 +628,170 @@ Provide only the HTML code without explanations. The HTML will be used with ZenM
+        {"accuracy": accuracy, "f1_score": 0.89},
+        infer_model=True  # Automatically finds pipeline's model
+    )
+```
+
+**Key features**
+* **Namespace organization** - group related pipelines, artifacts, and resources under a single entity
+* **Version tracking** - automatically version your ML solutions with each pipeline run
+* **Lineage management** - trace all components back to training pipelines, datasets, and code
+* **Stage promotion** - promote solutions through lifecycle stages (dev → staging → production)
+* **Metadata association** - attach any metrics or parameters to track performance over time
+* **Workflow integration** - connect training, evaluation, and deployment pipelines in a unified view
+
+**Common model operations**
+```python
+from zenml import Model
+from zenml.client import Client
+
+# Get all models in your project
+models = Client().list_models()
+
+# Get a specific model version
+model = Client().get_model_version("my_classifier", "latest")
+
+# Promote a model to production
+model = Model(name="my_classifier", version="v2")
+model.set_stage(stage="production", force=True)
+
+# Compare models with their metadata
+model_v1 = Client().get_model_version("my_classifier", "v1")
+model_v2 = Client().get_model_version("my_classifier", "v2")
+print(f"Accuracy v1: {model_v1.run_metadata['accuracy'].value}")
+print(f"Accuracy v2: {model_v2.run_metadata['accuracy'].value}")
+```
+
+**Best practices**
+* Create models with meaningful names that reflect the ML capability or business feature they represent
+* Use consistent metadata keys across versions for better comparison and tracking
+* Tag models with relevant attributes for easier filtering and organization
+* Set up model stages to track which ML solutions are in which environments
+* Use a single model entity to group all iterations of a particular ML capability, even when the underlying technical implementation changes
+
+Learn more: [Models | ZenML](https://docs.zenml.io/how-to/model-management-metrics/models)
+
+## 13 Create a parent Docker image for faster builds
+
+**Why** -- reduce Docker build times from minutes to seconds and avoid dependency headaches by pre-installing common libraries in a custom parent image. This approach gives you faster iteration cycles, consistent environments across your team, and simplified dependency management—especially valuable for large projects with complex requirements.
+
+```bash
+# 1. Create a Dockerfile for your parent image
+cat > Dockerfile.parent << EOF
+FROM python:3.9-slim
+
+# Install system dependencies
+RUN apt-get update && apt-get install -y --no-install-recommends \
+    git \
+    curl \
+    build-essential \
+    && rm -rf /var/lib/apt/lists/*
+
+# Install Python dependencies that rarely change
+RUN pip install --no-cache-dir \
+    zenml==0.54.0 \
+    tensorflow==2.12.0 \
+    torch==2.0.0 \
+    scikit-learn==1.2.2 \
+    pandas==2.0.0 \
+    numpy==1.24.3 \
+    matplotlib==3.7.1
+
```

Comment:
you also need to install stack requirements. here you can reuse some of the work you did in the tutorial you write for CI/CD